src/main.rs

```diff
@@ -99,6 +99,7 @@ fn create_app<'a, 'b>(
                 .help(
                     "Memory parameters \
                      \"size=<guest_memory_size>,file=<backing_file_path>,mergeable=on|off,\
+                     hotplug_method=acpi|virtio-mem,\
```

Comment:
@teawater You also need to extend the cloud-hypervisor.yaml `MemoryConfig` with the `hotplug_method` property, setting it to `Acpi` as default.

Reply:
Done.